tests/functionnal/Search.php

```diff
@@ -1633,6 +1633,36 @@ public function providerAddWhere()
                 'meta' => false,
                 'expected' => "  AND  (`glpi_users_users_id_tech`.`id` = '2') ",
             ],
+            [
+                'link' => ' AND ',
+                'nott' => 0,
+                'itemtype' => \Monitor::class,
+                'ID' => 11, // Search ID 11 (size field)
+                'searchtype' => 'contains',
+                'val' => '70',
+                'meta' => false,
+                'expected' => "  AND  (`glpi_monitors`.`size`  LIKE '%70.%%'  ) ",
```

Comment:
Is it normal to have 2 percent chars ?

Reply:
No. It was required to add the extra percent because the criteria was getting replaced with an equals comparison later if the value was numeric.